src/Microsoft.Azure.SignalR.Common/Utilities/ConnectionStringParser.cs

```diff
@@ -164,11 +167,11 @@ private static AccessKey BuildAadAccessKey(Uri uri, Dictionary<string, string> d
                 {
                     if (dict.TryGetValue(ClientSecretProperty, out var clientSecret))
                     {
-                        return new AadAccessKey(uri, new ClientSecretCredential(tenantId, clientId, clientSecret));
+                        return new AadAccessKey(uri, new ClientSecretCredential(tenantId, clientId, clientSecret), true);
```

Comment:
don't see AadAccessKey ctor change to support `true`, what is it?

Reply:
failed to revert the patch when rebasing, fixed it